polling_stations/apps/file_uploads/tests/test_check_pending.py

```diff
@@ -0,0 +1,49 @@
+from django.core import mail
+from django.conf import settings
+from django.test import TestCase
+from file_uploads.models import Upload
+from councils.models import Council
+from file_uploads.management.commands import check_pending
+from freezegun import freeze_time
+
+
+class TestCheckPending(TestCase):
+    @freeze_time("2021-04-01 12:06:00")
+    def test_handle_possible_failure(self):
+        self.council = Council.objects.create(
+            council_id="E09000001",
+            name="City of London Corporation",
+        )
+        Upload.objects.create(
+            gss=self.council,
+            election_date="2021-05-06",
+            timestamp="2021-04-01 12:00:00",
+        )
+        self.assertEqual(len(Upload.objects.all()), 1)
+        self.assertEqual(Upload.objects.first().warning_about_pending_sent, False)
+        qs = check_pending.Command().handle()
+        self.assertEqual(Upload.objects.first().warning_about_pending_sent, True)
+        self.assertEqual(len(mail.outbox), 1)
+        self.assertEqual(mail.outbox[0].subject, "Possible failed uploads")
+        self.assertEqual(
+            mail.outbox[0].body, f"A file(s) upload was not successful: {qs}"
+        )
+        self.assertEqual(mail.outbox[0].to, [settings.DEFAULT_FROM_EMAIL])
+
+    @freeze_time("2021-04-01 12:03:00")
+    def test_handle_nothing_pending(self):
```

Comment:
This name is confusing because there _is_ an upload pending, but it's within the allowed time limit.